Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs

```diff
@@ -105,8 +103,8 @@ private void OnValidate()
     {
         if (!modalScreenFactory.LandscapePrefab && !modalScreenFactory.PortraitPrefab)
         {
-            modalScreenFactory.LandscapePrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>("Packages/io.chainsafe.web3-unity.web3auth/Runtime/Prefabs/W3AConnectionScreen_L.prefab");
-            modalScreenFactory.PortraitPrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>("Packages/io.chainsafe.web3-unity.web3auth/Runtime/Prefabs/W3AConnectionScreen_P.prefab");
+            modalScreenFactory.LandscapePrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>(UnityEditor.AssetDatabase.GUIDToAssetPath("9e5f859444d8b4a448e79b28a6033fd7"));
```

Comment:
Yikes, why are we using GUID and not the path?

Reply:
It a safety measure. When you move/rename file its path changes, but GUID always stays the same.

Reply:
Ah fair, those things will never change their GUID.